src/status_im/signals/core.cljs

```diff
@@ -1,20 +1,25 @@
 (ns status-im.signals.core
-  (:require [status-im.chat.models.loading :as chat.loading]
-            [status-im.ethereum.subscriptions :as ethereum.subscriptions]
+  (:require [status-im.ethereum.subscriptions :as ethereum.subscriptions]
+            [status-im.i18n :as i18n]
             [status-im.mailserver.core :as mailserver]
+            [status-im.multiaccounts.login.core :as login]
             [status-im.multiaccounts.model :as multiaccounts.model]
-            [status-im.node.core :as node]
             [status-im.pairing.core :as pairing]
             [status-im.transport.filters.core :as transport.filters]
             [status-im.transport.message.core :as transport.message]
             [status-im.utils.fx :as fx]
             [status-im.utils.types :as types]
-            [taoensso.timbre :as log]
-            [status-im.multiaccounts.login.core :as login]))
+            [taoensso.timbre :as log]))
 
 (fx/defn status-node-started
-  [{db :db :as cofx} event]
-  (login/multiaccount-login-success cofx))
+  [{db :db :as cofx} {:keys [error]}]
+  (if (empty? error)
+    (login/multiaccount-login-success cofx)
+    {:db (-> db
+             (update :multiaccounts/login dissoc :processing)
+             (assoc-in [:multiaccounts/login :error] (if (= error "file is not a database")
```

Comment:
hm, is it temporary?

Reply:
what do you mean? that is the error that status-go returns when there is a wrong password. I am not aware of possible other errors at the moment so I figured in case this happens we should just show them